### DISCUSSION TITLE
Hover not working on border width

### DISCUSSION THREAD

**Discussion by amirsafaricg** [tailwindlabs/tailwindcss#7906] [Help]
` class="
          flex
          justify-center
          items-center
          text-2xl
          rounded-lg
          border-2
          border-blue-600
          hover:border-0
          bg-transparent
          hover:bg-blue-600 	
          text-blue-600
          hover:text-white
          h-16
          w-48
        "`
        
       Hi everyone, This is the class Im using for a button but whenever I hover on the button, The border should be 0 but still stay's at the value of 2 pixels 
       How can I fix it ?

**Comment by calmyournerves** [comment]
Hi! This seems to [work on Tailwind Play](https://play.tailwindcss.com/ACdYJonc3P). Maybe there is an issue with your setup or some other CSS interfering?
